controller/src/main/java/io/pravega/controller/server/rpc/auth/GrpcAuthHelper.java

```diff
@@ -75,7 +75,13 @@ public String checkAuthorization(String resource, AuthHandler.Permissions expect
 
     public String checkAuthorizationAndCreateToken(String resource, AuthHandler.Permissions expectedLevel) {
         if (isAuthEnabled) {
-            checkAuthorization(resource, expectedLevel);
+            try {
+                checkAuthorization(resource, expectedLevel);
+            } catch (RuntimeException e) {
```

Comment:
Would it have to be runtime or can we narrow the scope to a specific base exception?

Reply:
The Auth Handler implementation may be an externally supplied plugin and it might throw a RuntimeException (we don't wrap the handler at this point). Here, we log the exception, just in case it does that, to aid debugging.